ui/pages/routes/routes.component.js

```diff
@@ -242,6 +242,16 @@ export default class Routes extends Component {
     );
   }
 
+  onEditTransactionPage() {
```

Comment:
We only want to disable this when a transaction is being edited, not when it is first being created

Reply:
ah, you can use `getSendStage(state) === SEND_STAGES.EDIT` to determine this